index.html

```diff
@@ -1,252 +1,248 @@
+                        <img src="img/Snapshoot Portfolio(10).png" alt="snapshoot Portfolio picture">
+                    </div>
+                    <div class="project-card">
+                        <h2>Tonic</h2>
+                        <h3>CANOPY &nbsp; <img src="img/Counter.png" alt="dot symbol">&nbsp;<span> Back End Dev
+                                &nbsp; <img src="img/Counter.png" alt="dot symbol"> &nbsp; 2015 </span></h3>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                        <ul>
+                            <li class="html-tag">html</li>
+                            <li class="css-tag">css</li>
+                            <li class="js-tag">javascript</li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
                 </div>
-            </section>
+            </div>
 
-            <section class="canopy-third-card-section">
-                <div class="snapshoot-portofolio">
-                    <img src="img/Snapshoot Portfolio (2).png" alt="snapshoot-portofolio">
-                </div>
-                <div class="project-card">
-                    <h2>Tonic</h2>
-                    <h3>CANOPY &nbsp;<img src="img/Counter.png"  alt="dot symbol"> &nbsp;<span> Back End Dev &nbsp;<img src="img/Counter.png"alt="dot symbol"> &nbsp; 2015</span></h3>
-                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    <ul>
-                        <li>html</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button type="button">See Project</button>
+            <div class="canopy-card-section">
+                <div class="mycanopy-section">
+                    <div class="project-card">
+                        <h2>Multi-Post Stories</h2>
+                        <h3>FACEBOOK &nbsp; <img src="img/Counter.png" alt="dot symbol">&nbsp;<span> Full Stack Dev
+                                &nbsp; <img src="img/Counter.png" alt="dot symbol"> &nbsp; 2015 </span></h3>
+                        <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                        <ul>
+                            <li class="html-tag">html</li>
+                            <li class="ruby-tag">Ruby on rails</li>
+                            <li class="css-tag">css</li>
+                            <li class="js-tag">javascript</li>
+                        </ul>
+                        <button type="button">See Project</button>
+                    </div>
+                    <div class="snapshoot-portofolio">
+                        <img src="img/Snapshoot Portfolio(12).png" alt="snapshoot Portfolio picture">
+                    </div>
```

Comment:
**- You did a great job in your design but kindly make sure you use the appropriate path to display your image on my side the image is broken please fix that  :+1: 

## Your design 
![image](https://user-images.githubusercontent.com/95297251/221247008-ffc00842-d987-4823-a6c3-39246325bf4b.png)

**